src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -447,7 +447,9 @@ bool AccountStoreSC<MAP>::UpdateAccounts(const uint64_t& blockNum,
       // Check if the sender has enough balance to pay gasDeposit
       if (fromAccount->GetBalance() < gasDeposit + transaction.GetAmount()) {
         LOG_GENERAL(WARNING,
-                    "The account doesn't have enough gas to create a contract");
+                    "The account doesn't have enough gas to create a contract. Bal: "
```

Comment:
This is useful, but various stability reasons we like to keep the changes to AccountStoreSC.tpp to the bare minimum.    Makes sense to apply this change to Steve's PR: https://github.com/Zilliqa/Zilliqa/pull/2895

Reply:
steves PR has been merged already - do you want me to exclude this change from master?